web-frontend/src/pages/Assets/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './DeleteAssets';
+export * from './Assets';
+export * from './AssetsImportModal';
+export * from './AssetsList';
+export * from './AssetsLoading';
```

Comment:
```suggestion
export * from './AssetsLoading';

```

Prettier fix. 😉

Reply:
I don't see what the change is ? ://

Reply:
> I don't see what the change is ? :// but maybe attach a screen shot and I'll fix it manually

Run:

```shell
npm run prettier:fix
```

Reply:
it doesn't change anything for me

Reply:
run `npm run fmt`, that's the one executed on the CI